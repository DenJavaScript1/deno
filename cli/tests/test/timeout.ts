Deno.test("test 1", () => {
  throw new Error();
});
