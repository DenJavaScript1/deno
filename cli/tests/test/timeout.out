running 1 test from [WILDCARD]/test/timeout.ts
test test 1 ...
Exceeded timeout of [WILDCARD]. Use --timeout to increase the timeout value

test result: FAILED. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])
